source/UIAHandler/remote.py

```diff
@@ -26,20 +27,32 @@
 )
 
 
+_isSupported: bool = False
+
+
+def isSupported() -> bool:
+	"""
+	Returns whether UIA remote operations are supported on this version of Windows.
+	"""
+	return _isSupported
+
+
 def initialize(doRemote: bool, UIAClient: UIA.IUIAutomation):
 	"""
 	Initializes UI Automation remote operations.
-	This function is deprecated and will be removed in the future.
-	The following parameters are ignored:
+	The following parameters are deprecated and ignored:
 	@param doRemote: true if code should be executed remotely, or false for locally.
 	@param UIAClient: the current instance of the UI Automation client library running in NVDA.
 	"""

```

Comment:
I think it would be worth logging a warning if the parameters are set

Reply:
At the moment, they are always set, when called by `UIAHandler`. They were never optional arguments.